File: Java/I Międzyszkolny Fraktal.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        int sumOfStudents = 0;
        Scanner input = new Scanner(System.in);
        for (int i = 0; i < 4; i++) 
        {
            int allStudents = input.nextInt();
            int onlineStudents = input.nextInt();
            sumOfStudents += allStudents - onlineStudents;
        }
        input.close();
        System.out.println(sumOfStudents);
    }
}

File: C++/I Międzyszkolny Fraktal.cpp
#include <iostream>

int main() 
{
    int sumOfStudents = 0;
    for (int i = 0; i < 4; ++i) 
    {
        int allStudents, onlineStudents;
        std::cin >> allStudents >> onlineStudents;
        sumOfStudents += allStudents - onlineStudents;
    }
    std::cout << sumOfStudents << '\n';
    return 0;
}